frontend/src/components/WofPopper.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <div class="popper">
+        <div class="popper-content">
+            <slot></slot> 
+        </div>
+    </div>
+</template>
+
+<script>
+export default{
+    name: "WofPopper"
+}
+</script>
+
+<style lang="less">
+@import './common.less';
+
+.popper{
+    position: fixed;
+    z-index: 9998;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    background-color: rgba(0, 0, 0, 0.5);
+}
+
+.popper-content{
```

Comment:
Change name to "popper__content" and use nesting (look into Less documentation and examples in other components)